src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -355,7 +457,231 @@ TEST_CASE("concurrency_free_ctx_ids")
   // The first sequence should only be called two times, once at the very start,
   // and once during shutdown
   //
-  CHECK(trrm.stats_->sequence_status.seq_ids_to_count.at(1) == 2);
+  CHECK(tcm.stats_->sequence_status.seq_ids_to_count.at(1) == 2);
+}
+
+TEST_CASE("Concurrency - shared memory infer input calls")
+{
+  PerfAnalyzerParameters params{};
+  params.max_concurrency = 4;
+  bool is_sequence_model{false};
+
+  const auto& ParameterizeAsyncAndStreaming{[&]() {
+    SUBCASE("sync non-streaming")
```

Comment:
is streaming + sync illegal combination?

Reply:
Yes:

https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/perf_analyzer.cc#L150-L156